sdk/devcenter/developer-devcenter-rest/CHANGELOG.md

```diff
@@ -1,14 +1,13 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
-
-### Features Added
+## 1.0.0 (2023-05-23)
+This release updates the Azure DevCenter library to use the 2023-04-01 GA API.
 
 ### Breaking Changes
 
-### Bugs Fixed
-
-### Other Changes
+ - Environments client now works with "environment definitions" instead of "catalog items"
+ - Createing a new environment requires passing `environmentDefinitionName` instead of `catalogItemName`
+ - Createing a new environment requires passing an additional parameter `catalogName`
```

Comment:
```suggestion
 - Environments client now works with "environment definitions" instead of "catalog items"
 - Creating a new environment requires passing `environmentDefinitionName` instead of `catalogItemName`
 - Creating a new environment requires passing an additional parameter `catalogName`
```

Environments client - could you also mention the name of the client?